src/content/docs/extensions/full-text-search.md

```diff
@@ -47,22 +47,26 @@ The following optional parameters are supported:
 
 - `stemmer`: The text normalization technique to use. Should be one of: `arabic`, `basque`, `catalan`, `danish`, `dutch`, `english`, `finnish`, `french`, `german`, `greek`, `hindi`, `hungarian`, `indonesian`, `irish`, `italian`, `lithuanian`, `nepali`, `norwegian`, `porter`, `portuguese`, `romanian`, `russian`, `serbian`, `spanish`, `swedish`, `tamil`, `turkish`, or `none` if no stemming is to be used. Defaults to `english`,
 which uses a Snowball stemmer.
-
-The example below shows how to create an FTS index on the book table with the `abstract` and `title` properties using the `porter` stemmer.
+- `stopWords`: To prevent a full-text search index from becoming bloated, the full-text search engine omits commonly occurring string keywords(stopwords) when building and querying the full-text search index. By default, the built-in english stopwords is used. Stopwords list can be customized using the `stopWords` parameter. Kùzu accepts `stopWords` given in the following format:
+1. A node table with only a single string column of stopwords.
+2. A PARQUET/CSV file with only a single string column of stopwords. This file can be stored in cloud storage platforms like Amazon S3 or Google Cloud Storage (GCS) or made accessible via HTTPS. If hosted remotely, ensure the httpfs extension is enabled and valid credentials (e.g., access keys) are configured to authenticate and securely access the file.
```

Comment:
We should mention the fact that a header row is not required, something like this:
> A Parquet/CSV file with a single string column of stopwords (no header required).

Reply:
update the doc.